tests/console/openldap2.pm

```diff
@@ -0,0 +1,122 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test OpenLDAP 2 using a modified test suite from upstream
+# Maintainer: Alexandre Makoto Tanno <atanno@suse.com>
+
+use strict;
+use warnings;
+use base "consoletest";
+use testapi;
+use utils 'systemctl';
+use utils 'zypper_call';
+use version_utils 'is_sle';
+
+# Path to the test suite
+my $test_path = '/opt/openldap-tests/tests/';
+
+# List of backends to test
+my @backends_sle12 = qw(bdb hdb);
```

Comment:
This can be just `@backends` as there is no difference

Reply:
In fact there is a slight difference between @backends for SLE 12 and SLE 15. There are different tests for each version.

Reply:
You're right. Removing

Reply:
Done